catalog/README.md

```diff
@@ -0,0 +1,9 @@
+# Catalog
+
+This package holds i18n for DBaaS Controller.
+
+[gotext](https://pkg.go.dev/golang.org/x/text@v0.3.3/cmd/gotext?tab=doc)
+
+To merge translations and generate catalog use:
+
+    make gen
```

Comment:
It is not clear what file should be updated manually with translations, why we need two files, what should be done after translation changed, etc.